web/static/js/components/category_column.jsx

```diff
@@ -5,9 +5,9 @@ import styles from "./css_modules/category_column.css"
 
 function CategoryColumn(props) {
   const categoryToEmoticonUnicodeMap = {
```

Comment:
Check it!

[rocket-ship.svg.zip](https://github.com/stride-nyc/remote_retro/files/1072892/rocket-ship.svg.zip)